AzureDay.WebApp/AzureDay.WebApp.Notification/Email/Ext/RestorePassword.cs

```diff
@@ -0,0 +1,11 @@
+﻿using AzureDay.WebApp.Config;
```

Comment:
the whole file is redundant and should be removed, restore password will be handled by azure ad b2c